bapdoc.ml

```diff
@@ -27,6 +27,7 @@ let libraries = [
   [
     "bap-arm", "ARM", "ARM architecture";
     "bap-x86-cpu", "X86_cpu", "x86 family";
+    "bap-mips", "MIPS", "MIPS architecture";
```

Comment:
Can you remove it, please? 
here is said that there is a `bap-mips` library, what actually is not true.
So it will break our documentation.